src/membership/MembershipCard.tsx

```diff
@@ -61,18 +79,17 @@ const MembershipCard: React.FC<MembershipCardProps> = (
           </Link>
         </p>
         <p className="card-header-icon">
-          <span className="icon">
-            <i className="fas fa-angle-down" aria-hidden="true"></i>
-          </span>
+          <AdminTag isAdmin={membership.admin} />
         </p>
       </header>
       <div className="card-content">
         <div className="content">
-          <AdminTag isAdmin={membership.admin} />
+          Lorem ipsum about the org, maybe a link, not sure what might go here.
```

Comment:
Don't have to do it now, but it could be stats about the org. How many members, subscriptions, etc.

Reply:
Good call. I updated the placeholder text for now.